system/ee/installer/controllers/wizard.php

```diff
@@ -2249,7 +2249,7 @@ public function canRenameAutomatically()
 	 * Rename the installer
 	 * @return void
 	 */
-	private function rename_installer()
+	private function rename_installer($template_variables)
 	{
 		if (TRUE || ! $this->canRenameAutomatically())
```

Comment:
@intoeetive I believe this will always pass since it's TRUE OR?

Reply:
Good catch! I should put more attention into what I commit :(